src/js/themes/base.d.ts

```diff
@@ -22,7 +33,34 @@ export declare const generate: (
   scale?: number,
 ) => DeepReadonly<ThemeType>;
 
-type ExtendType = string | ((...args: any) => void);
+/**
+ * ExtendProps represents the props that will be provided to an ExtendType.
```

Comment:
I see how this all works and its great. 

i'm wondering if we can adjust the wording of the comments slightly around "value" and "type"...currently there is both

> ExtendType represents a valid value for `extend` in the theme

and 

> ExtendValue represents a valid `extend` value


I realize that this does make sense since an `ExtendType` may directly be an `ExtendValue`, but I still wonder if this can be clarified a bit. Perhaps changing 

`ExtendType` represents ~~a valid value~~ [the type] for `extend` values in the theme.

might be sufficient.

Reply:
Took your suggestion here and I think it's a little clearer. Thanks for the feedback!